modules/3d-tiles/docs/api-reference/tiles-3d-loader.md

```diff
@@ -144,7 +144,7 @@ After content is loaded, the following fields are guaranteed. But different tile
 | `cartographicOrigin` | `Number[3]`   | "Origin" in lng/lat (center of tile's bounding volume)                                                                                  |
 | `modelMatrix`        | `Number[16]`  | Transforms tile geometry positions to fixed frame coordinates                                                                           |
 | `attributes`         | `Object`      | Each attribute follows luma.gl [accessor](https://github.com/visgl/luma.gl/blob/master/docs/api-reference/webgl/accessor.md) properties |
-| `segmentationData`   | `Uint32Array` | An array of feature ids which specify which feature each vertex belongs to. Can be used for picking functionality.                      |
+| `objectIds`          | `Uint32Array` | An array of feature ids which specify which feature each vertex belongs to. Can be used for picking functionality.                      |
```

Comment:
Do we need to mention this in `docs/upgrade-guide.md`?

Reply:
Would we be able to document the loaded arrays in the https://loaders.gl/modules/tiles/docs/api-reference/tileset-3d

Reply:
Added to the `docs/upgrade-guide.md`

Reply:
Not clear where and how to document loaded arrays. Please provide more details on that.

Reply:
The basic idea is that all the tiled loaders should load into a common "normalized format". Ideally, for the application there should not be a difference between tiles loaded from 3D tiles, I3S, potree etc - the loaders should load into a common representation. 

But currently I see documentation in the specific loaders, such as https://loaders.gl/modules/i3s/docs/api-reference/i3s-loader#tile-content. I probably need to make a PR to reorganize this to make it clearer.

Reply:
https://github.com/visgl/loaders.gl/pull/1408